src/gui/MItem_network.cpp

```diff
@@ -177,6 +177,11 @@ MI_NET_IP::MI_NET_IP(NetDeviceID device_id)
     , device_id(device_id) //
 {
     index = netdev_get_ip_obtained_type(this->device_id());
+    if (index == NETDEV_STATIC) {
```

Comment:
Setting `index` should be enough for the menu item to display the value correctly. If it doesn't we need to investigate that.

Reply:
In this case, yes, as NETDEV_STATIC is Index 1, as long as index isn't this->index. Will try if it does work with just setting index.